doc/extensions/versioning_guidelines.md

```diff
@@ -0,0 +1,147 @@
+     - if only bug fixes, next version will be x.x.xbx+1
+   - if last version is stable as x.x.x (<1.0.0) or preview as x.x.xbx (< 1.0.0b1), next preview version should be 1.0.0b1
+
+### CLI Extension Version Transition Table
+
+Considering version increment rules above, version transition table can be summarized as table below:
+
+<table>
+    <tr>
+        <td rowspan="2">Next version tag</td>
+        <td rowspan="2">Last version</td>
+        <td colspan="3">Changes</td>
+        <td rowspan="2">Next version</td>
+    </tr>
+    <tr>
+        <td>Breaking change</td>
+        <td>Feature</td>
+        <td>Bug fix</td>
+    </tr>
+    <tr>
+        <td rowspan="4">Stable</td>
+        <td rowspan="3">2.0.0</td>
+        <td>&#10004</td>
+        <td>-</td>
+        <td>-</td>
+        <td>3.0.0</td>
+    </tr>
+    <tr>
+        <td>&#10006</td>
+        <td>&#10004</td>
+        <td>-</td>
+        <td>2.1.0</td>
+    </tr>
+    <tr>
+        <td>&#10006</td>
+        <td>&#10006</td>
+        <td>&#10004</td>
+        <td>2.0.1</td>
+    </tr>
+    <tr>
+        <td>No stable version</td>
+        <td>-</td>
+        <td>-</td>
+        <td>-</td>
+        <td>1.0.0</td>
+    </tr>
+    <tr>
+        <td rowspan="6">Preview</td>
+        <td rowspan="3">2.0.0</td>
+        <td>&#10004</td>
+        <td>-</td>
+        <td>-</td>
+        <td>3.0.0b1</td>
+    </tr>
+    <tr>
+        <td>&#10006</td>
+        <td>&#10004</td>
+        <td>-</td>
+        <td>2.1.0b1</td>
+    </tr>
+    <tr>
+        <td>&#10006</td>
+        <td>&#10006</td>
+        <td>&#10004</td>
+        <td>2.0.1b1</td>
+    </tr>
+    <tr>
+        <td rowspan="3">2.0.0b1</td>
+        <td>&#10004</td>
+        <td>-</td>
+        <td>-</td>
+        <td>3.0.0b1</td>
+    </tr>
+    <tr>
+        <td>&#10006</td>
+        <td>&#10004</td>
+        <td>-</td>
+        <td>2.0.0b2</td>
+    </tr>
+    <tr>
+        <td>&#10006</td>
+        <td>&#10006</td>
+        <td>&#10004</td>
+        <td>2.0.0b2</td>
+    </tr>
+</table>
+
+#### Notes
+- &#10004; means yes
+- &#10006; means no
+- \- means skip check
+- The major version number of preview can increase by at most one compared to the last stable version. For example, the last stable version os 2.0.0, if there are breaking changes for next preview release, the version will be 3.0.0b1. Then another breaking changes for the next preview release, the version will be 3.0.0b2
+- If previous version is less than 1.0.0, then, the next preview version should be 1.0.0b1 and the next stable one should be 1.0.0
+
+## Backward Compatibility of Extension Version Tags
+
+Previous extension module has a combination of semantic scheme and extra metadata tags (`azext.isExperimental` and `azext.isPreview`) to mark module release version. To keep backward compatibility:
+- `azext.isPreview` is reserved for all preview extension releases and `azext.isExperimental` is jointed into `azext.isPreview` for version tag simplicity. 
```

Comment:
How to understand this?
`azext.isExperimental` is jointed into `azext.isPreview` for version tag simplicity.

Reply:
`azext.isExperimental` is deprecated and will be replaced by azext.isPreview for version tag simplicity, since cli extension only supports two types of public releases now: stable and preview